spec/game_spec.rb

```diff
@@ -0,0 +1,35 @@
+require './lib/game'
+
+describe Game do
+  let(:game) { Game.new([1, 2, 3, 4], 'Daniel', 'Ronan') }
+
+  describe '#initialize' do
+    it 'Checks if the given player names are correct' do
+      expect(game.players).to eq('O' => 'Ronan', 'X' => 'Daniel')
+    end
+
+    it 'Checks if the game is still running' do
+      expect(game.still_playing).to be true
+    end
+
+    it 'Checks if the game is still running' do
+      expect(game.still_playing).not_to be false
+    end
+
```

Comment:
- [ ] It seems you have checked if the game is running twice. Kindly make one a negative test